### PR TITLE
Fixing the series of script so that PTF-analysis compiles

### PR DESCRIPTION
**Pull request by vgousyleblanc**
So, it looks like a redefinition of the PTF class was done but not propagated throught all the files creating compiling error. Fortunately, I did the modifications and now we should be back on track, the script compiles for me but I still have to test it to make sure everything works as expected ( The accelerometer script doesn't work for now but temperature reading and time does ). As discussed during the meeting, I am not 100% sure what caused these significiant different since even in the scripts I had modify, some modifications were done that I don't remember doing. Anyway, I will look at the history ! I apologize for this little delay and confusion !

### CODE REVIEW

File: acc_to_csv.cpp [hunk 1]
```diff
@@ -63,16 +64,23 @@ int main(int argc, char** argv) {
...
 
 	       auto reading  = wrapper.getReadingForPhidget(phidget);
 
 	       csv << reading.Ax[0] << "," << reading.Ay[0] << "," << reading.Az[0];
 
-	       if (phidget != 100) {
-	 	csv << ",";
+	       if (phidget == 100) {
+			   auto acceleration=wrapper.getReadingAcceleration();
+			    csv << acceleration.acc_x[0] << "," << reading.acc_y[0] << "," << reading.acc_z[0];
```
Comment (thomaslindner) [inline acc_to_csv.cpp]: This line doesn't compile for me.  Better would be 

               if (phidget == 100) {
                           auto acceleration=wrapper.getReadingAcceleration();
-                           csv << acceleration.acc_x[0] << "," << reading.acc_y[0] << "," << reading.acc_z[0];
+                           csv << acceleration.acc_x << "," << acceleration.acc_y << "," << acceleration.acc_z;
               }
             }

File: src/wrapper.cpp [hunk 2]
```diff
@@ -196,11 +211,17 @@ bool Wrapper::setDataPointers() {
     Time_1->SetAddress(&ti.time_c);
 	
 	
-    TBranch
-      *ACC_x= tree->GetBranch("gantry0_x"), *g0Y = tree->GetBranch("gantry0_y"), *g0Z = tree->GetBranch("gantry0_z"),
-        *ACC_y = tree->GetBranch("gantry0_rot"), *g0Phi = tree->GetBranch("gantry0_tilt"),
-      *ACC_z = tree->GetBranch("gantry1_x"), *g1Y = tree->GetBranch("gantry1_y"), *g1Z = tree->GetBranch("gantry1_z"),
-        *g1Theta = tree->GetBranch("gantry1_rot"), *g1Phi = tree->GetBranch("gantry1_tilt"),
+	 TBranch
```
Comment (thomaslindner) [inline src/wrapper.cpp:214]: This doesn't compile for me.  I don't think type of initialization is legal.  Better for me would be 

-        TBranch
-     *ACC_x= tree->GetBranch("phidgACC_Ax");
+     TBranch *ACC_x= tree->GetBranch("phidgACC_Ax");
         ACC_x->SetAddress(&ACC.acc_x);
-     *ACC_y= tree->GetBranch("phidgACC_Ay");
+     TBranch *ACC_y= tree->GetBranch("phidgACC_Ay");
         ACC_y->SetAddress(&ACC.acc_y);
-     *ACC_z= tree->GetBranch("phidgACC_Az");
+     TBranch *ACC_z= tree->GetBranch("phidgACC_Az");
         ACC_z->SetAddress(&ACC.acc_z);

**Comment by thomaslindner** [comment]
Did you forget to commit changes to src/wrapper.cpp?  I get link error:

/home/mpmtdaq/tmp/ptf-analysis/obj/acc_to_csv.o: In function `main':
/home/mpmtdaq/tmp/ptf-analysis/acc_to_csv.cpp:82: undefined reference to `Wrapper::getReadingAcceleration() const'
collect2: error: ld returned 1 exit status

Indeed, there is no Wrapper::getReadingAcceleration() in src/wrapper.cpp

**Comment by vgousyleblanc** [comment]
Yes, so everythings works except this inclusion of high resolution scan (I have been working on it for the last ~2h). I was perhaps too optimistic I thought I would be able to fix it without anyone checking. I am at the moment working on making this work, sorry for buzzing a bit too fast on this, give me ~1h and should compile for real this time

**Comment by vgousyleblanc** [comment]
So this pull request will know compile, I still haven't figure out what is the issue with including the high resolution accelerometer. However, I will probably do a separated pull request that will address these issues. Anyone can now at least use the ptf-analysis without having issues of compiling it

**Comment by thomaslindner** [comment]
OK, this compiles for me now.  Thanks for the changes.  This can be merged as far as I'm concerned now.

**Comment by pdeperio** [comment]
Thanks guys, sorry I don't have time to do a complete review right now, but will merge anyway.

**Pull request merged**